app/pages/project/index.cjsx

```diff
@@ -82,6 +87,20 @@ ProjectPage = React.createClass
       .then (pages) =>
         @setState {pages}
 
+  getSelectedWorkflow: (project, preferences) ->
+    @setState selectedWorkflow: 'PENDING'
+
+    preferredWorkflowID = preferences?.preferences.selected_workflow ? project.configuration?.default_workflow
+    if preferredWorkflowID?
+      apiClient.type('workflows').get preferredWorkflowID
```

Comment:
should be able to filter on active here as well as the workflow id if you want instead.